pystac/validation/local_validator.py

```diff
@@ -13,6 +15,48 @@
     from importlib.resources import files as importlib_resources_files
 
 VERSION = STACVersion.DEFAULT_STAC_VERSION
+
+
+def _read_schema(file_name: str) -> Dict[str, Any]:
+    with importlib_resources_files("pystac.validation.jsonschemas").joinpath(
+        file_name
+    ).open("r") as f:
+        return cast(Dict[str, Any], json.load(f))
+
+
+def get_local_schema_cache() -> Dict[str, Dict[str, Any]]:
+    return {
+        **{
+            (
+                f"https://schemas.stacspec.org/v{VERSION}/"
+                f"{name}-spec/json-schema/{name}.json"
+            ): _read_schema(f"stac-spec/v{VERSION}/{name}.json")
+            for name in ("item", "catalog", "collection")
+        },
+        **{
+            f"https://geojson.org/schema/{name}.json": _read_schema(
+                f"geojson/{name}.json"
+            )
+            for name in ("Feature", "Geometry")
+        },
+        **{
+            (
+                f"https://schemas.stacspec.org/v{VERSION}/"
+                f"item-spec/json-schema/{name}.json"
+            ): _read_schema(f"stac-spec/v{VERSION}/{name}.json")
+            for name in (
+                "basics",
+                "datetime",
+                "instrument",
+                "licensing",
+                "provider",
+            )
+        },
+    }
+
+
+############################### DEPRECATED #################################
+
 ITEM_SCHEMA_URI = (
     f"https://schemas.stacspec.org/v{VERSION}/item-spec/json-schema/item.json"
 )
```

Comment:
If this constant isn't deprecated, we should move it above the comment. If it is, we should deprecate it with `__getattr__`.

Reply:
Yeah I am trying to deprecate those constants. I didn't know about the `__getattr__` trick!